core/src/main/java/com/linecorp/armeria/internal/server/annotation/DefaultAnnotatedService.java

```diff
@@ -220,40 +223,53 @@ private static void warnIfHttpResponseArgumentExists(Type returnType,
         }
     }
 
-    public String serviceName() {
+    @Override
+    public String name() {
+        if (serviceName == null) {
```

Comment:
If you renamed `serviceName()` to `name()` then you should rename the field associated with it as well. `serviceName` could be renamed to `name`.

Reply:
I missed it. 
I modify `serviceName` -> `name`